subprojects/core/src/test/groovy/org/gradle/internal/fingerprint/classpath/impl/ClasspathFingerprintCompareStrategyTest.groovy

```diff
@@ -152,8 +152,8 @@ class ClasspathFingerprintCompareStrategyTest extends Specification {
 
         where:
         strategy   | includeAdded | results
-        CLASSPATH  | true         | [removed("three-old"), added("two-new"), removed("two-old"), added("three-new")]
-        CLASSPATH  | false        | [removed("three-old"), removed("two-old")]
+        CLASSPATH  | true         | [removed("three-old", "three"), added("two-new", "two"), removed("two-old", "two"), added("three-new", "three")]
+        CLASSPATH  | false        | [removed("three-old", "three"), removed("two-old", "two")]
```

Comment:
These now look weird. Can we make the difference between the two string parameters more obvious?

Reply:
I am now using a parameter map as for the fingerprints. This makes it clearer to me at least.

Reply:
Hm, this didn't make it more clear to me. By looking at the code the purpose of the two strings is perhaps even more obscure.

Reply:
Any suggestions you have in mind?